README.md

```diff
@@ -15,30 +15,97 @@ And you can check all of our open-source projects at [inaka.github.io](http://in
 
 ## Conventions & Rules
 
-#### Maintain existing style
-> When editing a system written by someone else, stick to the style in which it was designed. If a project has an overall style, stick to that when adding, removing or updating API endpoints as well.
+#### Use snake_case with JSON
+> When the endpoint returns a JSON, the keys should always be in snake_case, unless you have a super edge case.
```

Comment:
I found 3 misspelled words in this sentence: 
- snake_case
- JSON
	- *Suggested Replacements:* son, jason, jeon
- >
	- *Suggested Replacements:* , k, p, u, e, z, j, o, t, d, y, i, n, s, c, x, h, m, r, b, w, g, l, q, a, v, f